renderer/containers/Pay.js

```diff
@@ -25,6 +25,7 @@ const mapStateToProps = state => ({
   routes: state.pay.routes,
   maxOneTimeSend: channelsSelectors.maxOneTimeSend(state),
   walletBalanceConfirmed: balanceSelectors.walletBalanceConfirmed(state),
+  isProcessing: paySelectors.isQueryingRoutes(state),
```

Comment:
Can we keep the name as `isQueryingRoutes` rather than renaming here?

Reply:
i was thinking about this, but the i think this is the right way. it's a high level flag that can potentially aggregate more atomic states in it.

Reply:
Yeah that makes sense. Was thinking that right after I posted the comment 😀